lib/runtime-core/src/instance.rs

```diff
@@ -168,11 +168,9 @@ impl Instance {
         Rets: WasmTypeList,
     {
         let export_index =
-            self.module
-                .info
-                .exports
-                .get(name)
-                .ok_or_else(|| ResolveError::ExportNotFound {
+            &self
+                .get_export_index(name)
+                .map_err(|_| ResolveError::ExportNotFound {
```

Comment:
I believe all these `map_err(|_| ResolveError::ExportNotFound...` are redundant now because `get_export_index` is returning the same error.  Small simplification, but seems like we can drop them and just have `?`.

I like the new helper method!

Reply:
You right, i just need to change the `Result` type of `get_export_index` and we can used `?`.